### ISSUE TITLE
Data Sanitization - Twitter News Feed

### ISSUE THREAD

**Issue by Leundai** [Leundai/twanalyze#3]
# 🛑POSTPONED🛑

#### Description
Develop a function that strips away all the necessary information and only gets what will be needed. Specifically, this will be the tweet itself, the account name, number of retweets, and likes. Please ask in the discord chat if there are other important info that might be needed too.

This depends on #2 so make sure to collaborate with them so that you know what data will be expected. For now, however, make a JSON file that simulates what the [GET](https://developer.twitter.com/en/docs/twitter-api/v1/tweets/timelines/api-reference/get-statuses-user_timeline) request would look like. Also, this is relied on by #4

#### Files
References: https://developer.twitter.com/en

#### Tasks
- [x] Create a JSON file for testing purposes while #2 is being built
- [x] Build a function that strips away all the information that is needed and is either built back into a JSON file or is kept in memory to be sent as a GET request in Flask. 
- [ ] Collaborate with #2 once they have finished and test your function so that it works properly too.
